lease/lessor.go

```diff
@@ -68,9 +69,15 @@ type lessor struct {
 	idgen *idutil.Generator
 }
 
-func NewLessor(lessorID uint8, b backend.Backend, dr DeleteableRange) *lessor {
-	l := &lessor{
-		leaseMap: make(map[LeaseID]*lease),
+func NewLessor(lessorID uint8, b backend.Backend, dr DeleteableRange) *Lessor {
+	// ensure the most significant bit of lessorID is 0.
+	// so all the IDs generated by id generator will be greater than 0.
+	if int8(lessorID) < 0 {
```

Comment:
so it's OK if NewLessor(0x80,...) and NewLessor(0x0,...) alias to the same idgen?